models/article_edit.php

```diff
@@ -0,0 +1,213 @@
+<?php
+//###########################################
+// Model　記事編集画面
+//###########################################
+
+/**
+ * 記事編集画面出力に必要なパラメータの設定する
+ * また、記事の登録・削除の要求があった場合に処理を実行する判断を行う
+ */
+class Article_Edit extends Model{
+		protected $page_data = [
+		'article_id'=>'',
+		'title'=>'',
+		'body'=>'',
+		'title_errorMessage' => '',
+		'errorMessage' => '',
+		'css' => '<link rel="stylesheet" href="/css/article_edit.css">'
+	];
+
+	public function Action(){
+		// article_idが存在する場合、記事を取得
+		if(isset($_GET['article_id'])){
+			$this->page_data['article_id'] = $_GET['article_id'];
+			try{
+				// データベース接続
+				$pdo = DB_Connect::getPDO();
+				$stmt = $pdo->prepare('SELECT * FROM Articles WHERE article_id = :article_id');
+				$stmt->bindParam(':article_id',$_GET['article_id'], PDO::PARAM_INT);
+				$stmt->execute();
+				if ($row = $stmt->fetch(PDO::FETCH_ASSOC)) {
+					$this->page_data['title'] = $row['title'];
+					$this->page_data['body'] = $row['body'];
+				}
+			}catch(PDOException $e){
+				$this->error_message = 'データベースエラー:'.$e->getMessage();
+			}
+		}
+
+		$form = new ArticleEditForm();
+		//登録ボタン押下時の処理
+		if(isset($_POST['submit'])){
+			// 入力チェック
+			if($form->InputCheck()){
+				// 保存
+				$form->Save();
+			}
+			// ページ用のプロパティに代入
+			$this->page_data = [
+				'article_id'=>$form->getArticle_id(),
+				'title' => $form->getTitle(),
+				'body' => $form->getBody(),
+				'title_error_message' => $form->getTitleErrorMessage(),
+				'error_message' => $form->getErrorMessage(),
+				'css' => '<link rel="stylesheet" href="/css/article_edit.css">'
+			];
+		}
+
+		//削除ボタン押下時の処理
+		if(isset($_POST['delete'])){
+			// 削除
+			$form->Delete();
+			// 画面遷移
+			header("Location: /user.articles");
+		}
+
+		$this->page = new Page('記事編集画面', 'article_edit', $this->page_data);
+	}
+}
+
+/**
+ * 記事編集フォーム
+ */
+class ArticleEditForm{
+
+	protected $article_id;				// 記事ID
```

Comment:
ここで宣言している変数は、このクラス以外で使用していないので、privateにしましょう。
```suggestion
	private $article_id;				// 記事ID
```